app/services/reinstate_conditions_met.rb

```diff
@@ -25,7 +25,7 @@ def save
         ActiveRecord::Base.transaction do
           ApplicationStateChange.new(application_choice).reinstate_conditions_met!
           application_choice.update(attrs)
-          UpdateOfferConditions.new(application_choice: application_choice).call
+          application_choice.offer.set_conditions_to_met
```

Comment:
As this is a bulk update we should just use `update_all(status: 'met').

I don't think there is any value in adding custom methods to offer to perform this operation as it's already a one liner.

Reply:
Sounds good to me ill make the change